scenarios/nsp/service/reach_service_different_namespace.go

```diff
@@ -1,25 +1,22 @@
 package service
 
 import (
+	"context"
 	"errors"
 
 	"github.com/equinor/radix-cicd-canary/metrics"
 	nspMetrics "github.com/equinor/radix-cicd-canary/metrics/scenarios/nsp"
 	"github.com/equinor/radix-cicd-canary/scenarios/utils/config"
 	httpUtils "github.com/equinor/radix-cicd-canary/scenarios/utils/http"
-	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 )
 
-var logger zerolog.Logger
-
 // Reach tests that we are able to reach radix-canary-golang-prod endpoint
-func Reach(cfg config.Config, suiteName string) error {
-	logger = log.With().Str("suite", suiteName).Logger()
+func Reach(ctx context.Context, cfg config.Config) error {
 
 	client := httpUtils.GetHTTPDefaultClient()
 	url := "http://www.radix-canary-golang-prod:5000/health"
-	logger.Info().Str("url", url).Msg("requesting data")
+	log.Ctx(ctx).Info().Str("url", url).Msg("requesting data")
```

Comment:
```suggestion
	log.Ctx(ctx).Debug().Str("url", url).Msg("requesting data")
```
Other tests in the NSP suite logs the `requesting data` message as Debug. We shgould do the same here for consistency.